mubench.reviewsite/src/Controllers/Controller.php

```diff
@@ -19,4 +21,13 @@ public function __get($property)
         }
         return null;
     }
+
+    function getDetector($detector_id){
```

Comment:
I would prefer to have this on `Detector` as a static method `findByIdOrName()`.